docs/userguide/monitoring.rst

```diff
@@ -50,6 +59,8 @@ Install the visualization server::
    $ pip install git+https://github.com/Parsl/viz_server.git
 
 Once `viz_server` is installed, you can run the utility `parsl-visualize` in the directory with the
-monitoring.db sqlite file to launch a web page with the workflow visualization.
+`monitoring.db` sqlite file to launch a web page for the workflow visualization::
+
+   $ parsl-visualize sqlite:///<absolute-full-path-to-db>
```

Comment:
We should point out the absurd 4 slashes necessary in a note. Eg: `sqlite:////tmp/monitoring.db`.
Absolute-path is sufficient, we don't need to say absolute-full-path

Reply:
added new lines to explain that